src/pages/gallery/components/CreateCollection.tsx

```diff
@@ -6,6 +6,8 @@ import { CollectionAndItsLatestFile } from 'services/collectionService';
 import { getToken } from 'utils/common/key';
 import constants from 'utils/strings/constants';
 
+export const ERR_NO_INTERNET_CONNECTION = 1;
```

Comment:
Let's pull all of these constants into one file? Like how we have `errors.go` in museum?